packages/python-packages/api-stub-generator/apistub/nodes/_class_node.py

```diff
@@ -188,14 +214,27 @@ def _inspect(self):
 
             if self.is_enum and isinstance(child_obj, self.obj):
                 self.child_nodes.append(
-                    EnumNode(name=name, namespace=self.namespace, parent_node=self, obj=child_obj)
+                    EnumNode(
+                        name=name,
+                        namespace=self.namespace,
+                        parent_node=self,
+                        obj=child_obj
+                    )
+                )
+            elif inspect.isclass(child_obj):
+                self.child_nodes.append(
+                    ClassNode(
```

Comment:
What classes is this adding that we were missing before?

Reply:
Well, it snuck in because I had defined a decorator class inside another class and was surprised that it didn't appear. Then I realized we do our decorators as functions anyways and removed it. It would expose classes contained within another class, assuming they aren't prefixed with _. I'm not sure how common this is in Python, but it is common in Swift. I'm also fine to remove it.

Reply:
It's probably not common, but we'd want to see it if it happened - so I see no reason not to keep it :)